src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -565,6 +567,36 @@ Namespace Microsoft.CodeAnalysis.Semantics
             Return New LazyIsTypeExpression(operand, isType, _semanticModel, syntax, type, constantValue, isImplicit)
         End Function
 
+        Private Function CreateBoundLateInvocationOperation(boundLateInvocation As BoundLateInvocation) As IOperation
+            Dim expression As Lazy(Of IOperation) = New Lazy(Of IOperation)(Function() Create(boundLateInvocation.Member))
+
+            Dim isDynamicPropertyReference = boundLateInvocation.MethodOrPropertyGroupOpt?.Kind = BoundKind.PropertyGroup
+            Dim applicableSymbols As ImmutableArray(Of ISymbol)
+            If boundLateInvocation.MethodOrPropertyGroupOpt IsNot Nothing Then
+                applicableSymbols = If(isDynamicPropertyReference,
+                    DirectCast(boundLateInvocation.MethodOrPropertyGroupOpt, BoundPropertyGroup).Properties.As(Of ISymbol),
+                    DirectCast(boundLateInvocation.MethodOrPropertyGroupOpt, BoundMethodGroup).Methods.As(Of ISymbol))
+            Else
+                applicableSymbols = ImmutableArray(Of ISymbol).Empty
+            End If
+
+            Dim arguments As Lazy(Of ImmutableArray(Of IOperation)) = New Lazy(Of ImmutableArray(Of IOperation))(Function()
+                                                                                                                     Return If(boundLateInvocation.ArgumentsOpt.IsDefault,
+                                                                                                                        ImmutableArray(Of IOperation).Empty,
+                                                                                                                        boundLateInvocation.ArgumentsOpt.SelectAsArray(Function(n) Create(n)))
+                                                                                                                 End Function)
+            Dim argumentNames As ImmutableArray(Of String) = boundLateInvocation.ArgumentNamesOpt.NullToEmpty()
+            Dim argumentRefKinds As ImmutableArray(Of RefKind) = ImmutableArray(Of RefKind).Empty
```

Comment:
`BoundLateInvocation` node in VB doesn't seem to be storing the array of RefKinds for each argument. @AlekseyTs, is this expected or should I open a compiler bug?

Reply:
@AlekseyTs - We can do so by returning a default array for VB. Does that sound fine?